### PR TITLE
Fix typos in basics/structs

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: public/content/en/basics/structs.md [hunk 1]
```diff
@@ -28,7 +29,8 @@ is defined through a `this(...)` member function:
         }
             ...
 
-    Person p(30, 180);
+    Person p(30, 180); // initialization
+    t = Person(30, 180);  // creation
```
Comment (stonemaster) [inline public/content/en/basics/structs.md]: Shouldn't that be auto t?

Comment (wilzbach) [inline public/content/en/basics/structs.md]: T is already defined above, but I agree `auto` is clearer :)

Comment (stonemaster) [inline public/content/en/basics/structs.md]: Ah now I get it. We could make it clearer by chsnging the comment to "assignment to new instance"?

**Comment by wilzbach** [comment]
> Ah now I get it. We could make it clearer by chsnging the comment to "assignment to new instance"?

Done ;-)

**Pull request merged**